### ISSUE TITLE
About the New Update

### ISSUE THREAD

**Issue by alensa1337** [haxball/haxball-issues#1912]
When we open/close Chati, black 1-second-long lines appear. ///// It happens when we're not chatting, rarely

![giphy](https://user-images.githubusercontent.com/131692507/235510374-4bbe8515-ec0d-4bad-acfc-20425a2f6d08.gif)

**Comment by alensa1337** [comment]
> When we open/close Chati, black 1-second-long lines appear. ////// It happens when we're not chatting, rarely
> 
> ![giphy](https://user-images.githubusercontent.com/131692507/235510374-4bbe8515-ec0d-4bad-acfc-20425a2f6d08.gif) [ ](https://user-images.githubusercontent.com/131692507/235510374-4bbe8515-ec0d-4bad-acfc-20425a2f6d08.gif) ![giphy](https://user-images.githubusercontent.com/131692507/235510374-4bbe8515-ec0d-4bad-acfc-20425a2f6d08.gif) [ ](https://user-images.githubusercontent.com/131692507/235510374-4bbe8515-ec0d-4bad-acfc-20425a2f6d08.gif)

**Comment by basro** [comment]
I don't understand the problem you are describing is. What is a 1-second long line?

**Comment by alensa1337** [comment]
> I don't understand the problem you are describing is. What is a 1-second long line?



When we have the setting "Use low-latency canvas" in the Settings menu, which is the new feature you have introduced, we experience a decrease in FPS. When we open/close Chati, black 1-second-long lines appear. @basro

**Comment by basro** [comment]
What does it mean that a line is 1 second long? Do you mean the line lasts for 1 second?

In the video you've attached I don't see a line, the only odd thing I see is a flicker/green flash when opening the chat (but not when closing it).

What browser are you playing on? Are you using the --disable-frame-rate-limit option?

**Comment by alensa1337** [comment]
> What does it mean that a line is 1 second long? Do you mean the line lasts for 1 second?
> 
> In the video you've attached I don't see a line, the only odd thing I see is a flicker/green flash when opening the chat (but not when closing it).
> 
> What browser are you playing on? Are you using the --disable-frame-rate-limit option?

Yes yes, 1 millisecond black line. And yes i using disable frame limit.
